ci_build_pipelines/stages/build/steps/restlerfullbuild.yml

```diff
@@ -64,3 +64,7 @@ steps:
       artifactName: 'restler-fuzzer-drop-$(versionNumber)'
       publishLocation: 'Container'
 
+  - task: PublishPipelineArtifact@1
+    inputs:
+      targetPath: $(Agent.BuildDirectory)/restler_drop/
+      artifactName: restler-fuzzer-drop
```

Comment:
>restler-fuzzer-drop [](start = 20, length = 19)

Could you please name this 'restler-fuzzer-drop-for-docker'?  This makes it super clear that it's not the one to download if you just want a build of RESTler.

Reply:
Done.